repository-data/application/src/main/resources/hcm-content/content/documents/administration/bsh-labels/modules/events-listings-module.yaml

```
@@ -14,14 +14,15 @@
     hippostd:state: draft
     hippostdpubwf:createdBy: admin
     hippostdpubwf:creationDate: 2025-01-24T10:57:16.269Z
-    hippostdpubwf:lastModificationDate: 2025-02-21T15:36:17.373Z
+    hippostdpubwf:lastModificationDate: 2025-04-08T17:25:52.044+01:00
     hippostdpubwf:lastModifiedBy: admin
-    resourcebundle:descriptions: ['', '', '', '', '', '', '', '', '']
+    resourcebundle:descriptions: ['', '', '', '', '', '', '', '', '', '', '']
     resourcebundle:id: events-listings-module
     resourcebundle:keys: [filter, clear-all, sort-by, time, price, location, organizer,
-      registration, contact, apply]
+      registration, contact, apply, no-js]
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 26 but found 6

(indentation)